yarn-project/stdlib/src/event/event_metadata.ts

```diff
@@ -1,56 +0,0 @@
-import { z } from 'zod';
-
-import { type AbiType, AbiTypeSchema } from '../abi/abi.js';
-import { decodeFromAbi } from '../abi/decoder.js';
-import type { EventSelector } from '../abi/event_selector.js';
-import type { PublicLog } from '../logs/public_log.js';
-import { schemas } from '../schemas/index.js';
-import { L1EventPayload } from './l1_event_payload.js';
-
-/**
- * Represents metadata for an event decoder, including all information needed to reconstruct it.
- */
-export class EventMetadata<T> {
```

Comment:
What was this?

Reply:
It was a decoding utility but it felt largely useless so I nuked. Instead I just directly call decodeFromAbi:
<img width="1034" alt="image" src="https://github.com/user-attachments/assets/22c13b52-b715-4a84-acd9-14665f2894f1" />